src/main/java/com/minecolonies/coremod/entity/citizen/citizenhandlers/CitizenDiseaseHandler.java

```diff
@@ -56,7 +55,7 @@ public CitizenDiseaseHandler(final AbstractEntityCitizen citizen)
     @Override
     public void tick()
     {
-        if (citizen.getTicksExisted() % TICKS_20 == 0 && citizen.getCitizenColonyHandler().getColony().isActive() && !(citizen.getCitizenJobHandler()
+        if (citizen.getCitizenColonyHandler().getColony().isActive() && !(citizen.getCitizenJobHandler()
```

Comment:
Do we want to tick this so often?

Reply:
still ticks every 20, just called every 20 now

Reply:
ah ok